dissect/target/tools/shell.py

```diff
@@ -869,7 +869,7 @@ def cmd_enter(self, args: argparse.Namespace, stdout: TextIO) -> None:
 class UnixConfigTreeCli(TargetCli):
     def __init__(self, target: Target):
         TargetCmd.__init__(self, target)
-        self.config_tree = ConfigurationFs(target)
+        self.config_tree = ConfigurationFilesystem(target)
```

Comment:
`ls blah`

or any non-existing dir will not give an error is that okay?

Windows version gives:
`blah: no such subkey`

cd blah gives a very big error on the other hand, just a message would be fine.

Reply:
Ok, fine by me. I was just noticing the difference in behaviour.